internal/store/leaf/store_test.go

```diff
@@ -17,43 +17,28 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func createSubStore(dir string) (*Store, error) {
+func createSubStore(t *testing.T) (*Store, error) {
+	t.Helper()
+
+	dir := t.TempDir()
 	sd := filepath.Join(dir, "sub")
 
 	_, _, err := createStore(sd, nil, nil)
 	if err != nil {
 		return nil, err
 	}
 
-	if err := os.Setenv("GOPASS_CONFIG", filepath.Join(dir, ".gopass.yml")); err != nil {
-		return nil, err
-	}
-
-	if err := os.Setenv("GOPASS_HOMEDIR", dir); err != nil {
-		return nil, err
-	}
+	t.Setenv("GOPASS_CONFIG", filepath.Join(dir, ".gopass.yml"))
```

Comment:
Unrelated to this PR but the configs don't use yaml anymore for a while.
I'll need to change that.